javascript-practice/src/javascripts/utilities/url.js

```diff
@@ -0,0 +1,14 @@
+class Utilities {
```

Comment:
Should not define the utilities as a class

Reply:
https://www.geeksforgeeks.org/functional-programming-pure-and-impure-functions/

Reply:
I fixed it